src/session.ts

```diff
@@ -152,15 +152,29 @@ async function getUser({ ensureSignedIn = false } = {}) {
     return { user: null };
   }
 
-  const { sid: sessionId, org_id: organizationId, role } = decodeJwt<AccessToken>(session.accessToken);
+  const { sid: sessionId, org_id: organizationId, role, permissions } = decodeJwt<AccessToken>(session.accessToken);
+
+  const hasPermission = (permission: string) => {
```

Comment:
I don't think this is necessary since it's a pretty straightforward operation and just complicates the return object. Since `permissions` exists and is included in the return object it's trivial for developers to determine whether the user has a certain permission themselves.

Reply:
Sounds good, will remove!